File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.java
class Solution {
    public int maxProfit(int[] prices) {
        int max=0;
        for (int i=0;i<prices.length;i++){
            for (int j=i;j<prices.length;j++){
                if (i!=j){
                    int profit=prices[j]-prices[i];
                    if (profit>max){
                        max=profit;
                    }
                }
            }

        }

        return max;
        
    }
}

File: Data Structures & Algorithms/max-consecutive-ones-iii/submission-0.py
class Solution:
    def longestOnes(self, nums: List[int], k: int) -> int:
        zeroes=0
        maxcount=0
        start=0
        for i,el in enumerate(nums):
            if el==0:
                zeroes+=1
            while zeroes>k:
                if nums[start]==0:
                    zeroes-=1
                start+=1
            maxcount=max(i-start+1,maxcount)
        return maxcount
        